proposals/Exchange Server Load Balance proposal.md

```diff
@@ -0,0 +1,61 @@
+[TOC]
+
+## Motivation
+
+Load balancing across multiple application instances, here refers to Federated Learning jobs, is a commonly used technique for optimizing resource utilization, maximizing throughput, reducing latency, and ensuring fault‑tolerant configurations. In actual industrial use, we found that deploying only one exchange server will cause the data of multiple tasks to be forwarded through the single exchange server, resulting in lower performance and possible single point of failure. Therefore, we propose to add the load balancing mechanism of the exchange server on the original basis to avoid the above problems.
+
+## Goals
+
++ Solve the single point of failure of the exchange server
++ Improve the performance of running multiple tasks when using a single exchange server
+
+## Non-Goals
++ Solve the single point of failure of rollsite and its performance problems in each party.
+
+## UI or API
+Helm Charts `yaml` file
+
+~~~yaml
```

Comment:
Can it be like this:
```
...
module:
  - exchange
 
exchange:
  ip: xxx.xxx.xxx.xxx
  port: 9360
  fedlist:
     - name: party_1
       ip: xxx.xxx.xxx.xxx
       port: xxxx
    - name: party_2
       ip: xxx.xxx.xxx.xxx
       port: xxxx
```

In this case, the exchange just like a normal module, can be add to any FATE-cluster or standalone deployment. 

Alternatively, we can make exchange the same level of FATE or FATE_Serving, which means 
```
...
chartName: Exchange
chartVersion:  v1.5.0
```

Any comments? @jiahaoc1993 @owlet42 . I prefer the first option.

Reply:
I think the first approach is more natural and easy to understand